test/onnx_import.in.cpp

```diff
@@ -1712,3 +1713,49 @@ TEST(onnx_${BACKEND_NAME}, is_op_supported)
         });
     EXPECT_TRUE(onnx_import::is_operator_supported("AddQ", 1, "com.intel.ai"));
 }
+
+TEST(onnx_${BACKEND_NAME}, model_depth_to_space)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/depth_to_space.onnx"));
+
+    Inputs inputs;
+    inputs.emplace_back(std::vector<float>{
+        0.f, 1.f, 2.f, 3.f, 4.f, 5.f, 6.f, 7.f, 8.f, 9.f, 10.f, 11.f, 12.f, 13.f, 14.f, 15.f});
+
+    Outputs expected_output{std::vector<float>{
+        0.f, 4.f, 1.f, 5.f, 8.f, 12.f, 9.f, 13.f, 2.f, 6.f, 3.f, 7.f, 10.f, 14.f, 11.f, 15.f}};
+
+    Outputs outputs{execute(function, inputs, "${BACKEND_NAME}")};
+    EXPECT_TRUE(test::all_close_f(expected_output.front(), outputs.front()));
+}
+
+TEST(onnx_${BACKEND_NAME}, model_depth_to_space_chw)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/depth_to_space_chw.onnx"));
+
+    Inputs inputs;
+    inputs.emplace_back(std::vector<float>{
+        0.f, 1.f, 2.f, 3.f, 4.f, 5.f, 6.f, 7.f, 8.f, 9.f, 10.f, 11.f, 12.f, 13.f, 14.f, 15.f});
+
+    Outputs expected_output{std::vector<float>{
+        0.f, 4.f, 1.f, 5.f, 8.f, 12.f, 9.f, 13.f, 2.f, 6.f, 3.f, 7.f, 10.f, 14.f, 11.f, 15.f}};
+
+    Outputs outputs{execute(function, inputs, "${BACKEND_NAME}")};
+    EXPECT_TRUE(test::all_close_f(expected_output.front(), outputs.front()));
+}
+
+TEST(onnx_${BACKEND_NAME}, model_depth_to_space_bad_blocksize)
+{
+    EXPECT_THROW(onnx_import::import_onnx_model(file_util::path_join(
+                     SERIALIZED_ZOO, "onnx/depth_to_space_bad_blocksize.onnx")),
+                 std::runtime_error);
+}
+
+TEST(onnx_${BACKEND_NAME}, model_depth_to_space_no_blocksize)
+{
+    EXPECT_THROW(onnx_import::import_onnx_model(
```

Comment:
Please add a short comment explaining why these models are expected to throw an exception.